.github/workflows/daily-test.yml

```diff
@@ -0,0 +1,47 @@
+name: Run Tests Daily
+
+on:
+  schedule:
+    - cron:  '17 5 * * *'
```

Comment:
maybe add a comment that this is 5:17 UTC
i always forget how to read cron and it's nice to have a reminder

Reply:
good idea. added